opcua/server/history.py

```diff
@@ -4,6 +4,8 @@
 from opcua import Subscription
 from opcua import ua
 
+import sqlite3
+
```

Comment:
Can you move your sql backend to history_sql.py? so this file does not get too long, and we make iteven more  clear people can write their own.